views/sign-out.html

```diff
@@ -0,0 +1,57 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Are you sure you want to sign out? - {{ SERVICE_NAME }} - GOV.UK
+{% endblock %}
+
+{% block backLink %}
+  {{ govukBackLink({
+    text: "Back",
+    href: previousPage
+  }) }}
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <form method="post">
+
+      {{ govukRadios({
+        classes: "govuk-radios--inline",
+        idPrefix: "sign_out",
+        name: "sign_out",
+        fieldset: {
+          legend: {
+            text: "Are you sure you want to sign out?",
+            isPageHeading: true,
+            classes: "govuk-fieldset__legend--l"
+          }
+        },
+        hint: {
+            text: "Your answers will not be saved. You will need to start again if you want to register an overseas entity’s beneficial owner."
```

Comment:
Text here does not match that shown in the prototype image in the story. Guessing this is just the template from the prototype but need to check with analyst which is correct.

Reply:
During the refinement has been decided to change similar to confirmation statement. I have added a note on the story

Reply:
Ok. If agreed during the refinement then we need to ask analyst to put a note on the story 'Description' to say how the text varies. As it stands, delivering this will (should) result in a defect.

Reply:
The text change is odd though - you'd need to start again if you wanted to register an OE's beneficial owner OR, indeed, an Overseas Entity...

Reply:
Thanks Martin, I will rise it during the standup

Reply:
Karen has suggested a new text. It will be updated to `Your answers will not be saved. You will need to start again if you want to register an overseas entity and tell us about its beneficial owners.`

Reply:
Nice one.

Reply:
Updated